pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +91,57 @@ Our projects on Python Discord are open source and [available on Github](https:/
 
-[Guidelines](./contributing-guidelines/) - General guidelines you should follow when contributing to our projects.<br>
-[Style Guide](./style-guide/) - Information regarding the code styles you should follow when working on our projects.<br>
-[Review Guide](../code-reviews-primer/) - A guide to get you started on doing code reviews.
+  If you are here looking for the answer to a specific question, check out the sub-articles in the top right of the page to see a list of our guides.
 
-## Contributors Community
-We are very happy to have many members in our community that contribute to [our open source projects](https://github.com/python-discord/).
-Whether it's writing code, reviewing pull requests, or contributing graphics for our events, it’s great to see so many people being motivated to help out.
-As a token of our appreciation, those who have made significant contributions to our projects will receive a special **@Contributors** role on our server that makes them stand out from other members.
-That way, they can also serve as guides to others who are looking to start contributing to our open source projects or open source in general.
+### 1. Learn the basics of Git
+  Git is a _Version Control System_, software for carefully tracking changes to the files in a project. Git allows the same project to be worked on by people in different places. You can make changes to your local code and then distribute those changes to the other people working on the project.
 
-#### Guidelines for the @Contributors Role
+  Check out these [**resources to get started using Git**](./working-with-git/).
 
-One question we get a lot is what the requirements for the **@Contributors** role are.
-As it’s difficult to precisely quantify contributions, we’ve come up with the following guidelines for the role:
+### 2. Fork the repo
+  GitHub is a website based on the Git version control system that stores project files in the cloud. The people working on the project can use GitHub as a central place for sending their changes, getting their teammates' changes, and communicating with each other. Forking the repository that you want to work on will create a copy under your own GitHub account. You'll make your changes to this copy, then later we can bring them back to the PyDis repository.
 
-- The member has made several significant contributions to our projects.
-- The member has a positive influence in our contributors subcommunity.
+  Check out our [**guide on forking a GitHub repo**](./forking-repository/).
+### 3. Clone the repo
+  Now that you have your own fork you need to be able to make changes to the code. You can clone the repo to your local machine, commit changes to it there, then push those changes to GitHub.
 
-The role will be assigned at the discretion of the Admin Team in consultation with the Core Developers Team.
+  Check out our [**guide on cloning a GitHub repo**](./cloning-repository/).
```

Comment:
My immediate reaction to 1 is that it's a wall for people unfamiliar with git right off the bat. "Learn the basics" seems vague and a person's reaction might be that it's a daunting task they will need to undertake before even touching the specific project they want to work on, when really all that's expected is to use a few commands.

What I would consider doing here is, instead of making 1 a point of its own, to just make a preliminary note that development is being done with a versioning software called git, and from there link to our subpage with explanations on what git is and how to start out with it. Then we can maybe skip the explanations on what git is in 2 and 3.

Additionally, this might seem small, but 7 steps seems a bit long, I'd consider merging 2 and 3 just for that since they're both about github.

Reply:
I shortened/cut/merged the first 3 steps